hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/DefaultReplicationConfig.java

```diff
@@ -26,88 +26,53 @@
  */
 public class DefaultReplicationConfig {
```

Comment:
Should we also change a more accurate description for this class?

Reply:
Yes, I was considering that, but couldn't come up with any meaningful improvement.  Maybe next time.

Reply:
Oh, I mean the description of "Replication configuration for EC replication." 

Seems this is not only for EC replication?